src/Hazelcast.Net/Hazelcast.Net.csproj

```diff
@@ -5,6 +5,11 @@
     <RootNamespace>Hazelcast</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
The default language version for all our projects is specified in `src/Directory.Build.Props` so I think you want to update that file, not any of the `.csproj` files. Other than that... if the PR triggers no new warning then it's all good.